circle.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       # now install and checks the resulting file (just in case)
       - run:
           command: |
-            dnf install -y meson python3-devel pygobject3-devel
+            dnf install -y meson gettext python3-devel pygobject3-devel
```

Comment:
Done, even though for some reason it's listed in the wrong order here. [It shows correct in my branch](https://github.com/Hjdskes/piper/commits/fix/localization).